### DISCUSSION TITLE
puerts有哪些商业项目在使用

### DISCUSSION THREAD

**Discussion by landon30** [Tencent/puerts#399] [General]
最近比较关注puerts项目，想了解一下国内有哪些商业公司和商业项目在使用

**Comment by zitong** [comment]
我司新项目都切换到puerts了，因为确认好用

**Reply by marwie** [reply]
Could you elaborate a bit more which editors you mostly develop in (Unreal/Unity) and which platforms your project target? 
Do you internally have more tooling for using puerts in Unity?

**Comment by zombieyang** [comment]
最早公开的是台湾朋友 阿禾 的老虎机游戏。
期间有许多游戏使用，但他们碍于商业上的顾虑不会跟我们说。你可以在QQ群里问问。
webgl游戏则在 https://github.com/zombieyang/puerts_unity_webgl_demo 项目中有展示。
